.github/workflows/azure-deploy.yaml

```diff
@@ -104,6 +104,12 @@ jobs:
         run: |
           # fails if response status is not 2xx
           curl -f 'https://${{ env.AZURE_BACKOFFICE_NAME }}-staging.azurewebsites.net/${{ env.BACKOFFICE_HEALTHCHECK_PATH }}'
+          resp_status="$(curl --silent --head https://selfcare.io.pagopa.it/api/info | awk '/^HTTP/{print $2}')"
```

Comment:
**chore:** fix first curl execution

PS: suggestion not verified, checks it before accept! 😉 
```suggestion
          resp_status="$(curl --silent --head https://${{ env.AZURE_BACKOFFICE_NAME }}-staging.azurewebsites.net/${{ env.BACKOFFICE_HEALTHCHECK_PATH }} | awk '/^HTTP/{print $2}')"
```